cypress/integration/js2048Game.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
```

Comment:
Base on what I read this is needed for typescript. Is this needed for js?

Reply:
I use it for autocompletion